docs/source/configuration/in-memory-caching.mdx

```diff
@@ -42,6 +42,38 @@ supergraph:
         limit: 512
 ```
 
+### Cache warm up
+
+When loading a new schema, the query plan might change for some queries, so we cannot reuse that cache. If we invalidated the entire cache, this would incur latency on queries after the reload, because we would have to wait for them to be planned. To avoid that, the router takes the most used queries from the cache, and the entire list of persisted queries, generates query plans for them in advance and caches them before switching traffic over to the new schema.
+By default, the router warms up the cache with 30% of the queries already in cache, but it can be configured as follows:
```

Comment:
```suggestion
When loading a new schema, a query plan might change for some queries, so cached query plans cannot be reused. 

To prevent increased latency upon query plan cache invalidation, the Router warms will pre-compute query plans for:
* the most used queries from the cache
* the entire list of persisted queries.

Pre-computed plans will be cached before the Router switches traffic over to the new schema.

By default, the Router warms up the cache with 30% of the queries already in cache, but it can be configured as follows:
```